packages/gcf-utils/test/gcf-utils.ts

```diff
@@ -212,4 +214,98 @@ describe('GCFBootstrapper', () => {
       sinon.assert.calledOnce(configStub);
     });
   });
+
+  describe('getProbotConfig', () => {
+    let bootstrapper: GCFBootstrapper;
+    let secretClientStub: v1.SecretManagerServiceClient;
+    let secretsStub: sinon.SinonStub;
+    let secretVersionNameStub: sinon.SinonStub;
+
+    beforeEach(() => {
+      secretClientStub = new v1.SecretManagerServiceClient();
```

Comment:
I just realized that @alexander-fenster's refactor to make gRPC delay initialization made stubbing _way_ easier, this is great.

Reply:
Yep! It is so so so nice to use